funcbench/manifests/cluster.yaml

```diff
@@ -0,0 +1,12 @@
+projectid: {{ .PROJECT_ID }}
+zone: {{ .ZONE }}
+cluster:
+  name: {{ .CLUSTER_NAME }}
+  initialclusterversion: 1.14
```

Comment:
GKE regularly updates kubernetes engine, 1.14 is the least number of version it supports right now. Soon it will get obsolete. I think by default it will use the least version number even without setting this.